server/web/__init__.py

```diff
@@ -0,0 +1 @@
+from .setup import setup
```

Comment:
unused import. The import should be placed in a file, where the method is actually used.

Reply:
This is a way to reexport functions at the top level. This way you can write
```py
import web.setup
web.setup(app)
```
instead of
```py
import web.setup.setup(app)
```